wakusim.env

```diff
@@ -9,4 +9,10 @@ TRAFFIC_DELAY_SECONDS=15
 # Enable automatic Docker image updates.
 WATCHTOWER_ENABLED=true
 # Anvil RPC Node external IP and port
-RPC_URL=http://foundry:8545
\ No newline at end of file
+RPC_URL=http://foundry:8545
+# Contract-deployment
+PRIVATE_KEY=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80
```

Comment:
dont want to block this, but isn't this redundant? ETH_FROM and PRIVATE_KEY? I guess the ETH_FROM is the public address of ETH_FROM?

Reply:
it is required for the contract deployment because it is necessary for some upstream dependency (something about a known bug that hasn't been patched yet)